### PR TITLE
(#196) Add formula 5.6 from NEN-EN-1993.5:2008

### LINKED ISSUE #196: [✨ Feature request]: Add formula 5.6 from NEN-EN 1993-5
### Description

Add formula 5.6 from NEN-EN 1993-5

### Acceptance Criteria

- Formula 
- Latex function 
- Docstrings
- Unit-tests

### Dependencies

_No response_

### Confirmation

- [X] I have checked that this feature does not already exist in the issue tracker.
- [X] I have read and understood the contribution guidelines.

### PR DESCRIPTION
**Pull request by Klaassicking**
## Description

Add formula 5.6 from NEN-EN-1993.5:2008

$A_v = t_w \left(h - t_f \right)$

Fixes #196 

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+"""Eurocode NEN-EN 1993-5:2008."""
+
+NEN_EN_1993_5_2008 = "NEN-EN 1995-5:2008"
```
Comment (bro-wi) [inline blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/__init__.py:3]:
```suggestion
NEN_EN_1993_5_2008 = "NEN-EN 1993-5:2008"
```

File: blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_6.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+from blueprints.codes.eurocode.nen_en_1993_5_2008.chapter_5_ultimate_limit_states import NEN_EN_1993_5_2008
+from blueprints.codes.formula import Formula
+from blueprints.codes.latex_formula import LatexFormula
+from blueprints.type_alias import MM, MM2
+from blueprints.validations import raise_if_less_or_equal_to_zero
+
+
+class Form5Dot6ProjectedShearArea(Formula):
+    """Class representing formula 5.6 for the projected shear area."""
```
Comment (bro-wi) [inline blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_6.py:11]:
```suggestion
    """Class representing formula 5.6 for the projected shear area for each web of a U-profile or Z-profile."""
```

File: blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_6.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    label = "5.6"
+    source_document = NEN_EN_1993_5_2008
+
+    def __init__(
+        self,
+        h: MM,  # Overall height
+        tf: MM,  # Flange thickness
+        tw: MM,  # Web thickness
+    ) -> None:
+        """[Av] Calculate the projected shear area based on formula 5.6 from NEN-EN 1993-5:2007(E) art. 5.2.2(5).
```
Comment (bro-wi) [inline blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_6.py:22]: Shouldn't this be 2008?

File: blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_6.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    """Class representing formula 5.6 for the projected shear area."""
+
+    label = "5.6"
+    source_document = NEN_EN_1993_5_2008
+
+    def __init__(
+        self,
+        h: MM,  # Overall height
+        tf: MM,  # Flange thickness
+        tw: MM,  # Web thickness
```
Comment (bro-wi) [inline blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_6.py:20]:
```suggestion
        t_f: MM,  # Flange thickness
        t_w: MM,  # Web thickness
```

Try to keep the style consistent

File: blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_6.py [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    ) -> None:
+        """[Av] Calculate the projected shear area based on formula 5.6 from NEN-EN 1993-5:2007(E) art. 5.2.2(5).
+
+        Parameters
+        ----------
+        h : MM
+            [h] Overall height in [mm].
+        tf : MM
```
Comment (bro-wi) [inline blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_6.py:28]:
```suggestion
        t_f : MM
```

File: blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_6.py [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+            [tf] Flange thickness in [mm].
+        tw : MM
```
Comment (bro-wi) [inline blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_6.py:30]:
```suggestion
        t_w : MM
```

File: blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_6.py [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+            [tw] Web thickness in [mm].
+        """
+        super().__init__()
+        self.h: float = h
+        self.tf: float = tf
+        self.tw: float = tw
```
Comment (bro-wi) [inline blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_6.py:36]:
```suggestion
        self.t_f: float = t_f
        self.t_w: float = t_w
```

File: blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_6.py [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+    @staticmethod
+    def _evaluate(
+        h: MM,
+        tf: MM,
+        tw: MM,
```
Comment (bro-wi) [inline blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_6.py:42]:
```suggestion
        t_f: MM,
        t_w: MM,
```

File: blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_6.py [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+    ) -> MM2:
+        """Evaluates the formula for projected shear area."""
+        raise_if_less_or_equal_to_zero(h=h, tf=tf, tw=tw)
+        return tw * (h - tf)
```
Comment (bro-wi) [inline blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_6.py:46]:
```suggestion
        raise_if_less_or_equal_to_zero(h=h, t_f=t_f, t_w=t_w)
        return t_w * (h - t_f)
```

File: blueprints/codes/latex_formula.py
Comment (bro-wi) [inline blueprints/codes/latex_formula.py:1]: fracition not used in this PR, so shouln't be part of this PR.

File: tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/__init__.py [hunk 10]
```diff
@@ -0,0 +1 @@
+"""Module containing all formulas from NEN_EN_1993_5:2008: Chapter 5 - Ultimate limit states."""
```
Comment (bro-wi) [inline tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/__init__.py:1]:
```suggestion
"""Module containing all tests from NEN_EN_1993_5:2008: Chapter 5 - Ultimate limit states."""
```

File: tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/test_formula_5_6.py [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+
+        assert form == pytest.approx(expected)
+
+    @pytest.mark.parametrize(
+        ("h", "tf", "tw"),
+        [
+            (500, -20, 10),  # tf is negative
+            (500, 0, 10),  # tf is zero
+            (500, 20, -10),  # tw is negative
+            (500, 20, 0),  # tw is zero
```
Comment (bro-wi) [inline tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/test_formula_5_6.py:32]: 'h' zero and negative should be tested also

File: tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/test_formula_5_6.py [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+        ],
+    )
+    def test_raise_error_when_negative_or_zero_values_are_given(self, h: float, tf: float, tw: float) -> None:
+        """Test a zero and negative value for parameters h, tf, and tw."""
+        with pytest.raises(LessOrEqualToZeroError):
+            Form5Dot6ProjectedShearArea(h=h, tf=tf, tw=tw)
+
+    def test_latex_output(self) -> None:
```
Comment (bro-wi) [inline tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/test_formula_5_6.py:40]: use parametrize here too, as suggested by Sina in #191

File: tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/test_formula_5_6.py [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+        """Test the latex implementation."""
+        h = 500  # MM
+        tf = 20  # MM
+        tw = 10  # MM
+
+        form = Form5Dot6ProjectedShearArea(h=h, tf=tf, tw=tw)
+        assert form.latex().complete == r"A_v = t_w \left(h - t_f \right) = 10 \cdot \left(500 - 20 \right) = " + str(form)
+        assert form.latex().short == r"A_v = " + str(form)
+        assert str(form.latex()) == r"A_v = t_w \left(h - t_f \right) = 10 \cdot \left(500 - 20 \right) = " + str(form)
```
Comment (bro-wi) [inline tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/test_formula_5_6.py:49]:
```suggestion
        assert form.latex().complete == rf"A_v = t_w \left(h - t_f \right) = 10 \cdot \left(500 - 20 \right) = {form}"
        assert form.latex().short == rf"A_v = {form}"
        assert str(form.latex()) == rf"A_v = t_w \left(h - t_f \right) = 10 \cdot \left(500 - 20 \right) = {form}"
```

File: tests/codes/test_latex_formula.py
Comment (bro-wi) [inline tests/codes/test_latex_formula.py:1]: not part of this PR

**Review by bro-wi — COMMENTED** [review]
Nice work!

**Review by egarciamendez — APPROVED** [review]
Comments from Wichard have been implemented. Thanks! 🚀
